app/controllers/ems_common.rb

```diff
@@ -38,6 +38,10 @@ def show
       @lastaction = "show_timeline"
       tl_build_timeline                       # Create the timeline report
       drop_breadcrumb(:name => "Timelines", :url => show_link(@record, :refresh => "n", :display => "timeline"))
+    elsif @display == "dashboard"
+      @showtype = "dashboard"
+      @lastaction = "show_dashboard"
+      #drop_breadcrumb(:name => @ems.name + " (Dashboard)", :url => show_link(@ems)) ## Do we need a breadcrumb here?
```

Comment:
> Do we want a breadcrumb for a dashbaord?

@serenamarie125 please see @zeari question regarding breadcrumb.
imo - no, but it's up to you what is the right convention.